content/en/docs/tasks/administer-cluster/declare-network-policy.md

```diff
@@ -39,7 +40,7 @@ kubectl create deployment nginx --image=nginx
 deployment.apps/nginx created
 ```
 
-And expose it via a service.
+And expose it via a Service.
```

Comment:
Suggested change, line 43:
Expose the Deployment through a Service called `nginx`.

Reply:
Done.